antlr/src/main/antlr/FSH.g4

```diff
@@ -100,7 +101,7 @@ strength:           KW_EXAMPLE | KW_PREFERRED | KW_EXTENSIBLE | KW_REQUIRED;
 value:              STRING | MULTILINE_STRING | NUMBER | DATETIME | TIME | reference | canonical | code | quantity | ratio | bool | name;
 item:               name (KW_NAMED name)? CARD flag*;
 code:               CODE STRING?;
-concept:            STAR code (STRING | MULTILINE_STRING)?;
+concept:            STAR CODE+ STRING? (STRING | MULTILINE_STRING)?;
```

Comment:
Would it work to make `CODE` optional here? So `CODE*`. Because that way couldn't we eliminate `CaretValueRule` from `CodeSystem` altogether and process everything via `codeCaretValueRule`?

Reply:
I guess we couldn't remove `CaretValueRule` actually until the export side is done too, but does this idea make sense?

Reply:
I assume that you're talking about `codeCaretValueRule` here? So then those would be handled like this:
- if there are no `CODE` tokens, handle them the way we currently handle `caretValueRule` on a `CodeSystem`, with the bonus of knowing that the user didn't accidentally include a `path`.
- if there are `CODE` tokens, handle them using the new processing that applies values to concepts within the `CodeSystem`.

I think I like it, actually? It feels kind of weird, but at the same time, it sort of makes it clearer how we want folks to think about a `CodeSystem`: there are the elements on the `CodeSystem` itself that should be accessed with a `caretPath`, and there are the concepts that should be accessed with a `CODE+` path. The main side effect would be that a regular caret value rule would now be syntactically incorrect. I would want to make sure the error message in that case is still clear.